packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -118,6 +118,10 @@ typedef FocusOnKeyCallback = KeyEventResult Function(FocusNode node, RawKeyEvent
 /// was handled.
 typedef FocusOnKeyEventCallback = KeyEventResult Function(FocusNode node, KeyEvent event);
 
+/// Signature of a callback used by [FocusManager.addEarlyKeyEventHandler] and
+/// [FocusManager.addLateKeyEventHandler].
```

Comment:
Can you document the semantics of the argument and return value here?

Reply:
Done.